### DISCUSSION TITLE
Post-install setup: Error 504 when configuring database

### DISCUSSION THREAD

**Discussion by crawfordlong** [Kovah/LinkAce#425] [Support]
Trying to set up with docker and postgres (also nginx-proxy-manager providing ssl with the domain running through cloudflare). I have followed the special instructions for postgres (including running the CLI migration command). When I attempt the post-install setup, I get a 504 error when attempting to click through the database setup screen after a fairly long pause (15 seconds?). 

The only thing I'm seeing in the logs is a single line (please note I have manually replaced my domain name with $URL below):

172.30.0.1 - - [02/Apr/2022:18:38:32 +0000] "POST /setup/database HTTP/1.1" 499 0 "https://$URL/setup/database" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.60 Safari/537.36 Edg/100.0.1185.29" "xxx.xxx.xxx.xxx"

Thoughts re: what to try?

**Comment by Kovah** [comment]
What about the logs under /storage/logs?

**Reply by crawfordlong** [reply]
There's nothing in that directory, unfortunately.

**Reply by Kovah** [reply]
Is that directory writable? If not, please make it writable (0777 to be sure). Just for testing, you can change `APP_DEBUG=true` to `APP_DEBUG=false` to enable extended logging.
